packages/client/validation/Legitity.ts

```diff
@@ -37,9 +38,8 @@ class Legitity {
 
   max(len: number, msg?: string) {
     if (this.value !== undefined) {
-      // this.value.length gives us the count of UTF-16 units, so 🔥 has a length of 2
-      // Spreading the string into an array gives us the desired length in codepoints (characters): https://stackoverflow.com/a/54369605
-      const value = [...this.value]
+      // https://stackoverflow.com/a/46085147
+      const value = _.toArray(this.value)
```

Comment:
I lean towards using `Array.from` as it also accurately counts emoji length and would mean we wouldn't need to install the fairly chunky lodash library. 

One edge case though seems to be combined emojis, like 👪, which is really three emojis in one. For that, lodash seems like the better option. 

No strong preference from me!

Reply:
The black cat doesn't work either...
```
> Array.from('🐈‍⬛')
< (3) ['🐈', '‍', '⬛']
```

Reply:
Interesting! Black cat is actually multiple emojis too 😂